users/views.py

```diff
@@ -89,7 +89,7 @@ def get(self, request, *args, **kwargs):  # noqa  (too complex)
                     continue
                 if re.match(getattr(settings, 'SOCIAL_AUTH_SUOMIFI_CALLBACK_MATCH'), next_url) is None:
                     continue
-                m.login_url += '&amp;idp=' + m.provider_id
+                m.login_url += '&idp=' + m.provider_id
```

Comment:
What was this?

Reply:
Some kind of strange copy-paste error I would assume. But it worked because that was equivalent to "&amp&idp=". In a newer Django the semicolon is no longer considered a query parameter separator, so the resulting parameter name was "amp;idp". Which obviously didn't work anymore.